task/monorail-server/src/bsp/sidecar_a.rs

```diff
@@ -194,6 +194,7 @@ impl<'a, R: Vsc7448Rw> Bsp<'a, R> {
         self.phy_vsc8504_init()?;
         self.phy_vsc8562_init()?;
         self.vsc7448.configure_ports_from_map(&PORT_MAP)?;
+        self.vsc7448.configure_vlan_semistrict()?;
```

Comment:
How do we call `configure_vlan_strict()` instead? (Wondering if `configure_vlan_strict()` should be the default, with the `-lab` app.tomls setting a feature to switch to `configure_vlan_semistrict()`.)

Reply:
Right now, we don't call `configure_vlan_strict()` ever!

I suspect we want to keep using `semistrict` in **both** the regular and lab images for the moment; we'll be using the regular image for compliance, and I imagine folks may want technician port access to arbitrary SPs.